### ISSUE TITLE
App won't start when running on windows mapped drive.

### ISSUE THREAD

**Issue by jonande** [tagspaces/tagspaces#2077]
**Describe the bug**
When running tagspaces.exe from the c: drive on windows (following a normal installation) it runs normally. After copying all files to a mapped drive (z:) the tagspaces.exe file will not open. When running form cmd window you see the following error output:
```
Opening file: Z:\TagSpaces\TagSpaces.exe
[9288:0327/192623.414:ERROR:gpu_process_host.cc(985)] GPU process launch failed: error_code=18
start_ws:53398
Starting Tagspaces WS on Z:\TagSpaces\resources - pid (11400)
[9288:0327/192626.743:ERROR:gpu_process_host.cc(985)] GPU process launch failed: error_code=18
[9288:0327/192627.290:ERROR:gpu_process_host.cc(985)] GPU process launch failed: error_code=18
[9288:0327/192627.415:ERROR:gpu_process_host.cc(985)] GPU process launch failed: error_code=18
[9288:0327/192627.462:ERROR:gpu_process_host.cc(985)] GPU process launch failed: error_code=18
```

**To Reproduce**
1. Install normally
2. Copy entire install directory contents to mapped drive
3. Run exe file
4. See error output described above

**Expected behavior**
App runs normally

**Screenshots**
Listing of mapped drive contents:
![image](https://github.com/tagspaces/tagspaces/assets/64518233/673edf19-9ba5-4f17-9aad-1673711ba95f)

Error:
![image](https://github.com/tagspaces/tagspaces/assets/64518233/9261daf6-5565-4c33-9a5f-4dd4eb1ff155)

**Desktop Application:**
 - Operating System: Windows 10
 - Browser - NA
 - TagSpaces Version 5.7.11

**Additional context**
Trying to run from mapped drive to be used as a portable (--portable) installation across multiple computers.

**Comment by jonande** [comment]
I also tried running this using the network share directly (`\\wdmycloud\docs`) instead of the mapped drive with the same result:

```
PS Microsoft.PowerShell.Core\FileSystem::\\wdmycloud\docs\TagSpaces> .\TagSpaces.exe
PS Microsoft.PowerShell.Core\FileSystem::\\wdmycloud\docs\TagSpaces>
Opening file: \\wdmycloud\docs\TagSpaces\TagSpaces.exe
[29392:0328/151326.026:ERROR:gpu_process_host.cc(985)] GPU process launch failed: error_code=18
start_ws:49352
Starting Tagspaces WS on \\wdmycloud\docs\TagSpaces\resources - pid (31016)
[29392:0328/151329.481:ERROR:gpu_process_host.cc(985)] GPU process launch failed: error_code=18
[29392:0328/151329.965:ERROR:gpu_process_host.cc(985)] GPU process launch failed: error_code=18
[29392:0328/151330.075:ERROR:gpu_process_host.cc(985)] GPU process launch failed: error_code=18
[29392:0328/151330.122:ERROR:gpu_process_host.cc(985)] GPU process launch failed: error_code=18
```

**Comment by jonande** [comment]
I really want to be able to run TagSpaces on a network share so it can be a portable installation so 2 computers running the app on the same network have all the same tags, locations, etc. Is there a better way to do this if what I'm trying to do (run from network location) is not supported?

**Comment by sytolk** [comment]
Maybe using S3 location (like in https://demo.tagspaces.com ) is better option for you?

**Comment by jonande** [comment]
Does the S3 location gain you shared location and tag configuration in the same way that using tagspaces in `--portable` mode does?

**Comment by sytolk** [comment]
Yes you can install you local S3 server too ([minio](https://min.io/) [zenko](https://github.com/scality/cloudserver)) and this is doc to add cloud location in tagspaces https://docs.tagspaces.org/tutorials/s3-bucket-locations/#step-5---create-cloud-location-in-tagspaces
Next you can export and share your S3 configuration to other computers and all will access to the same location and tags. 
Tags configuration can be predefined in extconfig too https://docs.tagspaces.org/tutorials/tagspaces-web-wasabi#preparing-extconfigjs

**Comment by jonande** [comment]
Thanks for the suggestions! I had read about the `extconfig.js` option but this appears as though it is a static set of tags/locations. When I read through all the tagspaces documentation it appeared to me as though the `--portable` option is the only way to have the ability to modify (add/update) tags/locations on one computer and have them reflected on the other computer without any operations to copy them between installations. This is the main driver for this ask. I guess what I really need to know is if running the app from a network share should work (ie there is just something wrong with my setup) or if it is known that running from a network share will not work (and then I should give up on this installation plan)?

**Comment by jonande** [comment]
Thanks for the follow up. I saw the location based tags section and plan to use this. I think it is exactly what I want!
